gapic/templates/examples/feature_fragments.j2

```diff
@@ -25,44 +25,69 @@ There is a little, but not enough for it to be important because
 
 {# response handling macros #}
 
+{% macro sample_header(fileHeader, sample, callingForm) %}
```

Comment:
You're not using `fileHeader` here. I recommend we have a dummy placeholder as the value for now, and implement the logic here to render it.

Reply:
Done